docs/release-notes.md

```diff
@@ -1,130 +1,453 @@
+<br>
+
+:material-new-box: **Feature: Encrypting Senstive Data**
 
-:material-new-box: **Feature: Certificate Pruning**
+All sensitive data stored in the SQLite database is now encrypted using an encryption key. If not provided by the user, Console automatically generates this key, ensuring that sensitive information remains securely protected.
+<br>
 
-RPC-Go now cleans up old Wireless and TLS configs when adding new ones. This tweak should help keep things tidier when it comes to reconfiguring a device.
+:material-update: **Quality of Life Updates**
 
+We've made significant improvements and fixed issues across Power Actions, User Consent, KVM, SOL, Network, TLS, and Hardware information features.
 <br>
 
-:material-update: **Dependencies Update: Kubernetes Deployment Vault/Kong/Postgres**
+## Updates to Other component 
 
-We've updated the core dependencies for our default Kubernetes deployment. **If you are using Kong, the Kong secrets will require changes.** [See the `secrets.yaml` file to see the new required `labels:` field for Kong secrets.](https://github.com/open-amt-cloud-toolkit/open-amt-cloud-toolkit/blob/main/kubernetes/charts/secrets.yaml)
+:material-new-box: **Feature: RPC-Go Now Supports Encrypted Config**
 
-The default Azure Kubernetes Service (AKS) deployment has now switched from a Postgres Single Server to a Postgres Flex Server as the Azure EOL date of Single Server is quickly approaching.
+RPC-Go now supports activating and configuring Intel® AMT devices using encrypted configuration profiles (in `YAML` format) for enhanced security. The encryption key needed to decrypt the profile is shown only once during the export process in Console, so make sure to save it. Detailed steps can be found [here](./GetStarted/Enterprise/activateDevice.md#activate-device).
 
-## Community Contributions
 
-:material-party-popper: Big shout out to **Github user @webD97** for both of these contributions! :material-party-popper:
+:material-update: **Bug Fixes and Maintenance**
 
-:material-star: **Fix: RPC-Go on Linux Now Returns Correct FQDN and Hostname**
+Fixed minor bugs in RPS, MPS, Sample-Web-UI and RPC-Go. Updated dependencies across all components to the latest libraries.
 
-[Resolves a popular issue.](https://github.com/open-amt-cloud-toolkit/rpc-go/issues/189) Previously on Linux, RPC-Go was not properly differentiating between the device's FQDN and hostname when the device was configured with a short hostname and was returning a FQDN identical to the hostname.
 
-<br>
+!!! note "Note From the Team"
+    
+    While we're excited to release the BETA version, our journey towards the Console MVP (v1.0) continues. We’re actively listening to your feedback and working on feature requests and bug fixes to make Console even better. Rest assured, we will continue supporting and maintaining our Cloud components (MPS, RPS, and MPS-Router) alongside this effort.
```

Comment:
Long term vision is to consolidate MPS, and RPS into Console to reduce the complexity in deployment for either enterprise or cloud deployments. Until then, we will continue supporting and maintaining our Cloud components (MPS, RPS, and MPS-Router) alongside this effort.